strapi/strapi
# List custom fields/add howto btn

Add a how-to link in the customs fields tab.

# How to test 
1. Go to the Content-Type Builder.
2. Click on Add another field and then go to the "Custom" tab
3. There should be a "How to add custom fields" link in the bottom side 

Link is: https://docs.strapi.io/developer-docs/latest/getting-started/introduction.html
